app/templates/suppliers/view_signed_agreement.html

```diff
@@ -0,0 +1,104 @@
+{% import 'macros/toolkit_forms.html' as forms %}
+
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+Countersign {{ framework.name }} agreement for {{ supplier.name }}
+{% endblock %}
+
+{% block breadcrumb %}
+    {%
+    with items = [
+        {
+            "link": url_for('.index'),
+            "label": "Admin home"
+        },
+        {
+            "link": url_for('.list_agreements', framework_slug=framework.slug),
+            "label": "{} agreements".format(framework.name)
```

Comment:
Trying to `.format` a string with a unicode string breaks in Python 2. I think we should start using `"%s agreements" | format(framework.name)` for things like this, even though we don't have any non-latin characters in our framework names right now.